.github/workflows/publish.yml

```
@@ -0,0 +1,60 @@
+name: publish
+
+on:
+  push:
+    branches: [master]
+    tags: ['*']
+
+jobs:
+  publish-rockspec-scm-1:
+    if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

You could use a latest version of the GitHub Action: https://github.com/actions/checkout
```suggestion
      - uses: actions/checkout@v3
```

It doesn't look critical to me, be free to ignore it.

---

Updated.